blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -46,6 +46,11 @@ None
 Fixes
 =====
 
+- The sys check that checks tables for compatibility with future versions now
```

Comment:
This reads a bit difficult and I think the segments detail is not particularly useful. I'd rephrase this to something along the lines of "Fixed tables compatibility check to correctly indicate when tables need to be recreated in preparation for a CrateDB upgrade towards a major version" or something like that